debian/copyright

```diff
@@ -4,7 +4,7 @@ Source: http://www.openucx.org
 
 Files: *
 Copyright (c) 2014-2015      UT-Battelle, LLC. All rights reserved.
-Copyright (c) 2014-2015      NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2014-2015. ALL RIGHTS RESERVED.
```

Comment:
it became misaligned now

Reply:
Fixed by commit [df92afe](https://github.com/openucx/ucx/pull/8353/commits/df92afebb14b7e6ffe85647ba2c42d2988f3f481)